src/components/relayPiifields.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {graphql, useStaticQuery} from 'gatsby';
+
+const query = graphql`
+  query PiiFieldsQuery {
+    allPiiFieldPath {
+      nodes {
+        path
+      }
+    }
+  }
+`;
+
+function PiiField({field}) {
+  const asterisk_flag = field.endsWith('other');
+
+  return (
+    <React.Fragment>
+      <dt>
+        <code>{field}</code>
+        {asterisk_flag ? <span style={{marginLeft: '5px'}}>*</span> : null}
```

Comment:
Instead of doing an inline style here, it would be cool to use a styled component here instead. (you could take inspiration from `MetricType` in `relayMetrics.tsx`)